client/src/components/reservation/reservation.ts

```diff
@@ -0,0 +1,65 @@
+import styled, { keyframes } from "styled-components";
+import { pulse } from "react-animations";
+
+export const ReservationSection = styled.div`
+  height: 400px;
+  background-image: url(${require("../../assets/reservationImage.jpg")});
+  background-color: ${({ theme }) => theme.colors.primary};
+  background-position: 100% 20%;
+`;
+
+export const ReservationTitle = styled.h3`
+  font-size: ${({ theme }) => theme.fontSize.title};
+  color: ${({ theme }) => theme.colors.primary};
+  font-weight: 400;
+  line-height: 0.75;
+  margin-bottom: 60px;
+`;
+export const Form = styled.form`
+  width: 100%;
+`;
+export const ReservationInput = styled.input`
```

Comment:
Why does input have domain area of Reservation? We won't go to use inputs with the same styles in another forms? I suppose that we should have a bunch of styled form components which will be reused across the project.

Reply:
I've already done it